src/lib/viewers/controls/controls-layer/ControlsLayer.tsx

```diff
@@ -10,13 +10,14 @@ export type Helpers = {
 
 export type Props = {
     children: React.ReactNode;
+    forceShow?: boolean;
```

Comment:
Can we avoid having a separate prop for showing/hiding? It could get difficult to keep things synced up.